docs/headless-form/changeset/demo/base-demo.md

```diff
@@ -0,0 +1,56 @@
+```hbs template
+<HeadlessForm
+  class='space-y-4'
+  @data={{changeset this.data this.validations}}
+  @dataMode='mutable'
+  @onSubmit={{this.handleSubmit}}
+  @validate={{validate-changeset}}
+  as |form|
+>
+  <form.Field @name='name' as |field|>
+    <Form::InputField
+      @label='Name'
+      @hint='This is where you put your name'
+      name='name'
+      @value={{field.value}}
+      @onChange={{field.setValue}}
+    />
+    <field.Errors />
```

Comment:
I slacked you about releasing the new version of headless to better support this. I think we can add the "proper" way to do this as part of this PR, should hopefully be easy. Happy to pair again if you want!